docs/components/attention/Example.vue

```diff
@@ -1,40 +1,87 @@
 <script setup>
-import { ref } from 'vue';
-import { wAttention, wBox } from '@warp-ds/vue';
+import { ref } from 'vue'
+import { wAttention, wBox } from '@warp-ds/vue'
 
 const tooltipTarget = ref(null)
 const calloutTarget = ref(null)
 const popoverTarget = ref(null)
 
 const tooltipShowing = ref(false)
-const calloutShowing = ref(false)
+const calloutShowing = ref(true)
 const popoverShowing = ref(false)
 </script>
 
 <template>
   <div class="component space-y-16">
     <div>
-      <h3 class="h4">Callout</h3>
+      <h4>Callout</h4>
       <div class="flex items-center">
-        <w-box neutral ref="calloutTarget" @mouseenter="calloutShowing = true; target = $refs.calloutTarget" @mouseleave="calloutShowing = false">Hover over me</w-box>
-        <w-attention callout right :target-el="calloutTarget ? calloutTarget.$el : null" v-model="calloutShowing" class="ml-8">
-          <p>This is a callout</p>
+        <w-box
```

Comment:
I thought it was mentioned somewhere that we should avoid making non-interactive elements as targets to Attention, because we then need to tweak the tabbing to such elements with tab index. Probably there will mostly be buttons that those components point to in real life. Wdyt?

Reply:
You're right! I'll remove it! :)